src/index.html

```diff
@@ -11,7 +11,149 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          name="Surname"
+          id="surname">
+        </div>
+        <div class="input">
+          <label for="name">Name:</label>
+          <input
+          type="text"
+          name="Name"
+          id="name">
+        </div>
+        <div class="input">
+          <label for="number">How old are You?</label>
+          <input
+            type="number"
+            value="12"
+            max="100"
+            name="age"
+            id="number"
+          >
+        </div>
+        <div class="input">
+          <label for="date">Full date of birth</label>
+          <input
+            type="date"
+            name="date"
+            id="date">
```

Comment:
```suggestion
          <input
            type="date"
            name="date"
            id="date"
          >
```
The closing bracket should be on the new line